README.md

```diff
@@ -75,14 +84,101 @@ free strings, even though there were not explicitly malloc'd. The sample C
 code shows examples.
 
 #### DPDK Sample Image
+
 The initial problem `app-netutil` is trying to solve is to collect initial
 configuration data for a DPDK application running in a container. The DPDK
 Library is written in C, so there is a sample Docker Image that leverages
 the C APIs of `app-netutil` to collect the initial configuration data an
 then use it to start DPDK. See:
 * [dpdk_app](samples/dpdk_app/dpdk-app-centos/README.md)
 
+### Custom Settings
+
+app-netutil is a set of library functions that are intended to be pulled
+into a container workload to collect system and network information. In
+order to tailor the implementation to match the workload's needs, there
+are a few settings that can be configured.
+
+For C, there is a single C API method to configure app-netutil configuration
+fields:
+* `SetConfig()`
+  * This function takes an input structure and applies the configured
+  fields. Since this C API is only writing (passing data in), no data is
+  allocated on the GO side and no special freeing of memory is required,
+  other than the C side needs to free and memory it specifically allocates.
+Since the base code is written in GO, the GO code can call the configuration
+APIs directly, so there is not one single GO configuration API. 
+
+#### Logging
+
+app-netutil supports four levels of logging:
+* **"error"**
+* **"warning"**
+* **"info"**
+* **"debug"**
```

Comment:
nit: line 118 and 119 are showed in the same line when displaying the rich diff, maybe add an empty line in between?